libraries/adaptive-expressions/src/expressionFunctionsForNode.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ExpressionEvaluator } from './expressionEvaluator';
+import { ExpressionType } from './expressionType';
+import { ExpressionFunctions } from './expressionFunctions';
+import { ReturnType, Expression } from './expression';
+import * as os from 'os';
+
+/**
+ * Inject Node-specific functions into the common builtin-function list.
+ */
+export class ExpressionFunctionsForNode {
```

Comment:
Given that the _initialize is executed immediately, the injectedFunctions is only called once, and the _ prefix indicates a private method others shouldn't call - I wonder if this needs to be exported or could be an immediately executed function instead?